network/p2p/middleware.go

```diff
@@ -409,7 +410,9 @@ func (m *Middleware) processMessage(msg *message.Message) {
 func (m *Middleware) Publish(msg *message.Message, channel network.Channel) error {
 
 	// convert the message to bytes to be put on the wire.
+	p := binstat.EnterTime("~3net:wire<4message2protobuf", "")
 	data, err := msg.Marshal()
+	binstat.LeaveVal(p, int64(len(data)))
```

Comment:
do we want to keep binstat always on for this method?

Reply:
@ramtinms Please see discussion in above / same comment.

Reply:
got it